iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/models/DeploymentServiceModel.java

```diff
@@ -84,7 +84,13 @@ public DeploymentServiceModel(Configuration deployment) throws
             this.packageType = PackageType.deviceConfiguration;
         }
 
-        this.configType = deployment.getLabels().get(ConfigurationsHelper.CONFIG_TYPE_LABEL.toString());
+        if (deployment.getLabels().containsKey(ConfigurationsHelper.CONFIG_TYPE_LABEL.toString())) {
```

Comment:
If you can empty in the else block then this file doesn't need to change, i think.